### ISSUE TITLE
Unable to add tags to notes via API

### ISSUE THREAD

**Issue by bbgth** [babybuddy/babybuddy#488]
With the new tags functionality implemented, I am trying to tag a note that is submitted through iOS Shortcuts (using Baby Buddy's API). When I do that, I get this response back from the server once I have done the POST

`{"id":97,"tags":["TestTag"],"child":1,"note":"Test notes","time":"2022-06-22T10:16:28.607573+00:00"}`

When I look it up in the notes on the website, I only see the notes (Test notes) and not the tag. I have to manually edit the note and tap on the tag named TestTag to tag the notes.

Am I doing something wrong here with my POST command?

POST command:
```
Method: POST
URL: https://babybuddy.url/api/notes/
Headers: Authorization: Token <token>
Request Body: JSON
- child: 1 (type: number)
- note: "Test notes" (type: string)
- tags: ["TestTag"] (type: array, array content string)
```

**Comment by cdubz** [comment]
@bbgth I have confirmed this isn't working. Thanks for reporting!

**Comment by cdubz** [comment]
This is fixed in the latest version v1.11.2. Thanks, again!
